repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/ratelimiting/CheckRateLimitWConfig.groovy

```diff
@@ -98,8 +77,8 @@ class CheckRateLimitWConfig extends ReposeValveTest {
         ["X-PP-Groups": "multi-limits"]     | ["X-PP-User": "multilimits"] | multilimit
         ["X-PP-Groups": "query-limits"]     | ["X-PP-User": "querylimits"] | querylimit
         ["X-PP-Groups": "unlimited"]        | ["X-PP-User": "unlimited"]   | unlimitedlimit
-        //**defect on get limit for multi group config issue REP-2233
-        //["X-PP-Groups": "user"]             | ["X-PP-User": "default"]     | defaultlimit
+        //This one should also pass, but it's being affected by a state bug: REP-2233
+        ["X-PP-Groups": "user"]             | ["X-PP-User": "default"]     | defaultlimit
```

Comment:
Ah yes. It isn't a problem any more. 

On October 21, 2015 2:48:18 PM CDT, William Scheidegger <notifications@github.com> wrote:
>> @@ -98,8 +77,8 @@ class CheckRateLimitWConfig extends ReposeValveTest
>{
>>          ["X-PP-Groups": "multi-limits"]     | ["X-PP-User":
>"multilimits"] | multilimit
>>          ["X-PP-Groups": "query-limits"]     | ["X-PP-User":
>"querylimits"] | querylimit
>>          ["X-PP-Groups": "unlimited"]        | ["X-PP-User":
>"unlimited"]   | unlimitedlimit
>> -        //**defect on get limit for multi group config issue
>REP-2233
>> -        //["X-PP-Groups": "user"]             | ["X-PP-User":
>"default"]     | defaultlimit
>> +        //This one should also pass, but it's being affected by a
>state bug: REP-2233
>> +        ["X-PP-Groups": "user"]             | ["X-PP-User":
>"default"]     | defaultlimit
>
>If this test is now fixed, then shouldn't the comment be removed?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/rackerlabs/repose/pull/1398/files#r42672560

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.